sdk/jobs/automl-standalone-jobs/automl-image-classification-multiclass-task-fridge-items/automl-image-classification-multiclass-task-fridge-items.ipynb

```diff
@@ -349,7 +349,7 @@
     "\n",
     "When using AutoML for image tasks, you need to specify the model algorithms using the model_name parameter. You can either specify a single model or choose to sweep over multiple models. Please refer to the [documentation](https://docs.microsoft.com/en-us/azure/machine-learning/how-to-auto-train-image-models?tabs=CLI-v2#configure-model-algorithms-and-hyperparameters) for the list of supported model algorithms.\n",
     "\n",
-    "## 3.1. Configure the job through the image_classification() factory function\n",
+    "## Configure the job through the image_classification() factory function\n",
```

Comment:
Could you please add a small section on set_image_model() parameters: We need not list all the parameters supported. Instead, a small description of it should be good. Maybe we can use the description from this part of the docs: https://docs.microsoft.com/en-us/azure/machine-learning/how-to-auto-train-image-models?tabs=CLI-v2#fixed-settings